### PR TITLE
Added `inline_text()` method for `tbl_ae()`

### PR DESCRIPTION
**Pull request by ddsjoberg**
@shannonpileggi this still needs some attention from me. But before I work more on it, what do you think of the user experience? Can it be improved?

### CODE REVIEW

**Review by shannonpileggi — COMMENTED** [review]
This looks really great!!

One thought I had is about the `ae_or_soc` argument. I don't have strong feelings either way, but I was wondering if the argument could/should just be named `row` for simplicity? Or would that make it more confusing?

In addition, it would be good to add ore documentation (can be a separate issue!) showing how to use `show_header_names()`, and that the stat column names are `+1` due to the hidden column (right?).

File: tests/testthat/test-inline_text.R [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      tbl_ae(
+        id = patient_id,
+        ae = adverse_event,
+        by = grade,
+        header = "**Grade {level}**"
+      ) %>%
+      inline_text(ae_or_soc = "Anaemia", column = stat_6),
+    "3 (30)"
+  )
+
```
[the remark this answers is not in the document]
Comment (ddsjoberg) [reply tests/testthat/test-inline_text.R:13]: added!

[the remark this answers is not in the document]
Comment (ddsjoberg) [reply tests/testthat/test-inline_text.R:13]: added

**Comment by ddsjoberg** [comment]
> One thought I had is about the `ae_or_soc` argument. I don't have strong feelings either way, but I was wondering if the argument could/should just be named `row` for simplicity? Or would that make it more confusing?

I update the argument name

> In addition, it would be good to add ore documentation (can be a separate issue!) showing how to use `show_header_names()`, and that the stat column names are `+1` due to the hidden column (right?).

There is an example of this in the in the help file for inline text. is there something more you're referring to?


Otherwise, i think this may be ready to go!

**Comment by shannonpileggi** [comment]
Thanks! You're right, it is there in the example.  And that usage guide is awesome!

 I was just thinking in the docs/vignette somewhere a quick explanation of why `stat_6` corresponds to grade 5, as that might not be intuitive to the user.

Otherwise agreed, this is good to go!

**Review by shannonpileggi — APPROVED** [review]

**Pull request merged**